checkstyle.xml

```diff
@@ -27,14 +28,12 @@
   </module>
 
   <module name="TreeWalker">
-    <!--<property name="cacheFile" value="${checkstyle.cache.file}"/>-->

```

Comment:
This is enabled in other projects, but I don't know what sets the cache file (something in the Maven plugin?).